orion/pkg/tests_integration/class_integration_test.go

```diff
@@ -0,0 +1,493 @@
+package integration_tests
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+	"time"
+)
+
+// Test: Create 3 Classes and GetAll()
+func Test_CreateClasses(t *testing.T) {
+	resetTable(t, domains.TABLE_CLASSES)
+
+	now := time.Now().UTC()
+	class1 := createClass(
```

Comment:
I think this may not work because in order to create a class, you must first create a program, a semester, and a location in order for the foreign keys to work.
In the unit tests, you didn't have to do any of this because it was all mocked. With an integration test, there's a real MySQL DB working in the background.

So looks like you'll probably need helper function(s) to create a program, a semester, and a location.